src/bp_ges/main.cpp

```diff
@@ -59,14 +66,20 @@ int main()
 
     // Get temperature data
     bit32 temperatureLHAA, temperatureRHAA;
+    uint8_t thermistorOverTemperatureLHAA, thermistorOverTemperatureRHAA;
+
     temperatureLHAA.f = temperatureSensorLHAA.read();
     temperatureRHAA.f = temperatureSensorRHAA.read();
+    thermistorOverTemperatureLHAA = ptcThermistorLHAA.read();
+    thermistorOverTemperatureRHAA = ptcThermistorRHAA.read();
```

Comment:
It is better to initialize the variables directly when you assign them. Costs one line less and avoids multi declarations
```suggestion
    uint8_t thermistorOverTemperatureLHAA = ptcThermistorLHAA.read();
    uint8_ thermistorOverTemperatureRHAA = ptcThermistorRHAA.read();
```
I notice that the temperature variables already does this, however, that is a union and cannot be initialized like this.